packages/preview-middleware/src/base/flp.ts

```diff
@@ -155,14 +150,39 @@ export class FlpSandbox {
         this.logger.debug(`Configured apps: ${JSON.stringify(this.templateConfig.apps)}`);
     }
 
+    /**
+     * Get the configuration for the developer mode.
+     *
+     * @param ui5MajorVersion - the major version of UI5
+     * @returns the configuration for the developer mode
+     * @private
+     */
+    private getDeveloperModeConfig(ui5MajorVersion: number): Map<string, string> {
```

Comment:
I think this function would look simpler structured like
```
if (ui5MajorVersion < 2) {
   return new Map([ ... ]);
} else {
  return new Map([ ... ]);
}
```

Reply:
ok, will do. Had it like this before and switch to this approach after consulting AI but I agree that having it explicitly written will increase the readability.

Reply:
done in https://github.com/SAP/open-ux-tools/pull/2598/commits/c8d6e8569dd5da189ca1c518e1c75e3136a97dca